docs/excel/custom-functions-json.md

```diff
@@ -130,9 +130,8 @@ The `options` object enables you to customize some aspects of how and when Excel
-|  `requiresAddress`  | boolean | No <br/><br/>Default value is `false`. | <br /><br /> If true, your custom function can access the address of the cell that invoked your custom function. To get the address of the cell that invoked your custom function, use context.address in your custom function. For more information, see [Determine which cell invoked your custom function](/office/dev/add-ins/excel/custom-functions-overview#determine-which-cell-invoked-your-custom-function). Custom functions cannot be set as both streaming and requiresAddress. When using this option, the 'invocationContext' parameter must be the last parameter passed in options. |
-|  `stream`  |  boolean  |  No<br/><br/>Default value is `false`.  |  If `true`, the function can output repeatedly to the cell even when invoked only once. This option is useful for rapidly-changing data sources, such as a stock price. If you use this option, Excel will call the JavaScript function with an additional `caller` parameter. (Do ***not*** register this parameter in the `parameters` property). The function should have no `return` statement. Instead, the result value is passed as the argument of the `caller.setResult` callback method. For more information, see [Streaming functions](custom-functions-web-reqs.md#streaming-functions). |
+|  `cancelable`  |  boolean  |  No<br/><br/>Default value is `false`.  |  A cancelable function must declare an `invocation` parameter as the last parameter in the body of the function. If `"cancelable": true`, Excel calls the `invocation.onCanceled` method which you specify in the body of the function. (Do ***not*** register the `invocation` parameter in the `parameters` property). For more information, see [Streaming and cancelable functions](custom-functions-web-reqs.md#streaming-and-cancelable-functions). |
+|  `stream`  |  boolean  |  No<br/><br/>Default value is `false`.  | If `true`, the function can output repeatedly to the cell even when invoked only once. This option is useful for rapidly-changing data sources, such as a stock price. A streaming function must also declare an `invocation` parameter as the last parameter in the body of the function. (Do ***not*** register the `invocation` parameter in the `parameters` property). Do not return a value within the body of the function. Instead, the result value is passed as the argument of the `invocation.setResult` callback method. For more information, see [Streaming and cancelable functions](custom-functions-web-reqs.md#streaming-and-cancelable-functions). |
```

Comment:
Consider "the function can output repeatedly" -> "the function repeatedly outputs"